templates/editEvents.html

```diff
@@ -164,17 +164,14 @@ <h3 class="panel-title">
                       <div id="modalReallyRemoveRound_{{eventCode}}" class="modal fade">
                         <div class="modal-dialog">
                           <div class="modal-content">
-                            <div class="modal-header">
-                              <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Cancel</span></button>
-                              <h4 class="modal-title">
-                                Are you sure you want to delete {{eventName eventCode}}: {{roundName lastRoundCode}}?
-                              </h4>
-                            </div>
+                            {{#cmHeader}}
+                              Are you sure you want to delete {{eventName eventCode}}: {{roundName lastRoundCode}}?
+                            {{/cmHeader}}
                             <div class="modal-body">
                               <p>It appears that this round contains results. If you delete it, those results will be lost.</p>
                             </div>
                             <div class="modal-footer">
-                              <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                              {{> cmCancelButton}}
```

Comment:
I had a Hamletian struggle with that too. I probably got it wrong.

`cm` is meant to be short for `ccmModal`. Because `ccmModalCancelButton` it too wordy, and `ccmCancelButton` doesn't properly convey that it's part of the `ccmModal` "package".

Of course the `cm` prefix doesn't convey anything either unless I can show up in person and give this mini-lecture to everyone reading the code.

So... I'm OK with `ccm`, unless you have an even better suggestion. When you look it up you see what it's for.